app/views/posts/index.html.erb

```diff
@@ -5,6 +5,7 @@
             <%= link_to 'Add new Image', new_post_path, id:'new_path', remote:true, :class=>"btn btn-sm btn-warning" %>
             <hr>
             <%= render @posts %>
+            <%= render @post %>
```

Comment:
change this. give understandable variable names

Reply:
changed